.github/workflows/release.yml

```
@@ -33,6 +33,9 @@ jobs:
   deploy-storybook:
     name: Deploy Storybook to GH Pages
     runs-on: ubuntu-latest
+    environment:
+      name: github-pages
+      url: ${{steps.deployment.outputs.page_url}}
```

actionlint (1.7.4)

property "deployment" is not defined in object type {deploy: {conclusion: string; outcome: string; outputs: {page_url: string}}}

(expression)